crt_portal/cts_forms/forms.py

```diff
@@ -963,6 +963,49 @@ class Meta:
         }
 
 
+class TemplateActions(Form, ActivityStreamUpdater):
```

Comment:
Maybe an opportunity here for a clearer name since `Template` is a thing in Django itself, what about `ReponseActions`?